webview/src/experiments/components/emptyState/GetStarted.tsx

```diff
@@ -1,8 +1,24 @@
-import React from 'react'
-import { Welcome } from './Welcome'
+import React, { useEffect } from 'react'
 import { AddColumns } from './AddColumns'
 import { EmptyState } from '../../../shared/components/emptyState/EmptyState'
+import { redirectToSetup } from '../../util/messages'
 
-export const GetStarted: React.FC<{ showWelcome: boolean }> = ({
-  showWelcome
-}) => <EmptyState>{showWelcome ? <Welcome /> : <AddColumns />}</EmptyState>
+export const GetStarted: React.FC<{ noExpsToDisplay: boolean }> = ({
+  noExpsToDisplay
+}) => {
+  useEffect(() => {
+    if (noExpsToDisplay) {
+      redirectToSetup()
+    }
+  }, [noExpsToDisplay])
+
+  return (
+    <EmptyState>
+      {noExpsToDisplay ? (
+        <span>No experiments to display. Redirecting to setup...</span>
```

Comment:
Could also leave the text empty since the redirect should happen quickly and users likely won't have time to see the message.